src/stratis_cli/_parser/_parser.py

```diff
@@ -70,50 +107,22 @@ def gen_parser():
     :rtype: ArgumentParser
     """
     parser = argparse.ArgumentParser(
-       description="Stratis Storage Manager",
-       prog='stratis'
+        description="Stratis Storage Manager",
```

Comment:
fixed, added to readme. I'm assuming what pep8 calls continuation lines are what you're referring to as non-semantic.